assets/helpers/contributions.js

```diff
@@ -498,7 +454,7 @@ function getContributionTypeRadios(
     accessibilityHint: 'Make a regular annual contribution',
   };
 
-  return annualTestVariant === 'annual' || annualTestVariant === 'annualHigherAmounts'
+  return annualTestVariant === 'control' || annualTestVariant === 'annualAmountsA'
```

Comment:
Since this test has just two variants, does this mean that with this test, Annual Contributions will be present for everyone, regardless of test participation?

Because there's actually a ticket to make Annual Contributions always present on the old flow, before launching the new flow test (because we don't want to compare the new flow with Annual vs the old flow half the time with Annual and half the time without). https://trello.com/c/lWta59KJ/108-npf-annual-contributions-to-100-audience-on-existing-flow.

So it would be a good thing if this is achieved as part of this PR! But we could probably make a few tweaks so that fact is clearer, e.g. remove this ternary condition if it can never evaluate to the right-hand side

Reply:
yes, all users will see the annual button.
If we're not going back, and don't need `notintest`, then you're right I can simplify some logic here

Reply:
Yes, the model is now: we always have three contribution types (one-off, monthly, annual) which have standard amounts, and we can override the amounts if the user is in some test variant. I **think** (though you could check with Jesse) that you can assume that for any amounts test the control has the standard amounts. It may be that after a successful test we update these standard amounts, affecting both the normal site behaviour and the control for future tests (as in this case). But I don't think we would ever have different control amounts for different tests running at the same time, or have a control which deviated from the normal site behaviour. So amounts tests are always about overriding amounts in the variants.

Anything which makes that clearer in code is helpful. But doesn't necessarily have to be in this PR